t/proxyustats.t

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use FindBin qw($Bin);
+use lib "$Bin/lib";
+use Carp qw(croak);
+use MemcachedTest;
+use IO::Select;
+use IO::Socket qw(AF_INET SOCK_STREAM);
+
+# TODO: possibly... set env var to a generated temp filename before starting
+# the server so we can pass that in?
+my $configfile = "/tmp/proxyustats.lua";
+
+if (!supports_proxy()) {
+    plan skip_all => 'proxy not enabled';
+    exit 0;
+}
+
+# Set up some server sockets.
+sub mock_server {
```

Comment:
can omit this stuff since it's unused?

Reply:
removed, thanks for the insight.

Reply:
for what it's worth I was trying to add some of these functions to the test library, but the test library can't run tests easily. I need to revisit that so we don't have to carry them in every file. It's not like that because I'm a mess it's just low priority.

Reply:
I don't see that as an urgent or blocking issue TBH. As we have more files, the pattern will be clear and then we can take action on refactoring.